realm/realm-jni/src/io_realm_internal_TableQuery.cpp

```diff
@@ -38,7 +37,7 @@ inline bool query_valid(JNIEnv* env, Query* pQuery)
 
 inline bool query_col_type_valid(JNIEnv* env, jlong nativeQueryPtr, jlong colIndex, DataType type)
 {
-    return TBL_AND_COL_INDEX_AND_TYPE_VALID(env, TQ(nativeQueryPtr)->get_current_table().get(), colIndex, type);
+    return TBL_AND_COL_INDEX_AND_TYPE_VALID(env, Q(nativeQueryPtr)->get_table().get(), colIndex, type);
```

Comment:
I think it is b/c TableQuery is to be removed. In TQ, we have

```c++
realm::TableRef get_current_table() {
    realm::TableRef table = get_table();

    // Go through the stack of subtables to find current subtable (if any)
    size_t size = subtables.size(); 
    for (size_t i = 0; i < size; ++i) {
        size_t index = subtables[i];
        table = table->get_subtable(index, 0);
    }
    return table;
}
```

<sup>*</sup>There is _finding current table_ logic